### PR TITLE
Added commands to pass spec argument

### PR DESCRIPTION
**Pull request by amarnathk1547**
Added commands to pass spec file as an argument so that users can run only the specs they wish to.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,15 +12,28 @@ Start chromedriver
 chromedriver --port=4444 --url-base=/wd/hub --verbose
 ```
 
+Let the terminal be open and start an other one.
```
Comment (WillBrock) [inline README.md]: Can you change this to:

Let the terminal be open and start another one.

File: README.md [hunk 2]
```diff
@@ -12,15 +12,28 @@ Start chromedriver
...
+
+To run each test individually, pass spec file path as parameter
+
+```
+./node_modules/.bin/wdio wdio.conf.js --spec=tests/check-title.test
+./node_modules/.bin/wdio wdio.conf.js --spec=tests/form-authentication.test
+./node_modules/.bin/wdio wdio.conf.js --spec=tests/mocha-chai.test
+./node_modules/.bin/wdio wdio.conf.js --spec=tests/selectors.test
+./node_modules/.bin/wdio wdio.conf.js --spec=tests/waitfor-commands.test
+```
```
Comment (WillBrock) [inline README.md]: Can you remove the = after the --spec. It's not needed.

```
./node_modules/.bin/wdio wdio.conf.js --spec tests/check-title.test
./node_modules/.bin/wdio wdio.conf.js --spec tests/form-authentication.test
./node_modules/.bin/wdio wdio.conf.js --spec tests/mocha-chai.test
./node_modules/.bin/wdio wdio.conf.js --spec tests/selectors.test
./node_modules/.bin/wdio wdio.conf.js --spec tests/waitfor-commands.test
```

**Review by WillBrock — CHANGES_REQUESTED** [review]
Thanks for the PR!

**Comment by amarnathk1547** [comment]
Modified as suggested

**Pull request merged**